src/docbuild/logging.py

```diff
@@ -67,15 +66,7 @@
 }
 
 def create_base_log_dir(base_log_dir: str | Path = BASE_LOG_DIR) -> Path:
-    """Create the base log directory if it doesn't exist.
-
-    This directory is typically located at :file:`~/.local/state/docbuild/logs`
-    as per the XDG Base Directory Specification.
-
-    :param base_log_dir: The base directory where logs should be stored.
-        Considers the `XDG_STATE_HOME` environment variable if set.
-    :return: The path to the base log directory.
-    """
+    """Create the base log directory if it doesn't exist."""
```

Comment:
What's wrong with the parameter descriptions? 😉 Please re-add them or change it if we need to adapt them:


```suggestion
    """Create the base log directory if it doesn't exist.
    
    This directory is typically located at :file:`~/.local/state/docbuild/logs`		
    as per the XDG Base Directory Specification.

    :param base_log_dir: The base directory where logs should be stored.
        Considers the `XDG_STATE_HOME` environment variable if set.
    :return: The path to the base log directory.
    """
```

Reply:
> What's wrong with the parameter descriptions? 😉 Please re-add them or change it if we need to adapt them:

Mistakenly removed it. Added it back.